packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -9,6 +9,8 @@ import Animated, {
 	useSharedValue,
 	withDelay,
 	withTiming,
+	ZoomInEasyDown,
+	ZoomOutEasyDown,
```

Comment:
We use the pre-defined layout effects for producing the animations, you can check how they look like in the documentation:
- [Zoom enter animations](https://docs.swmansion.com/react-native-reanimated/docs/api/LayoutAnimations/entryAnimations#zoom)
- [Zoom exit animations](https://docs.swmansion.com/react-native-reanimated/docs/api/LayoutAnimations/exitAnimations#zoom)

Reply:
TIL about those! So cool!